src/index.ts

```diff
@@ -1,6 +1,6 @@
-import fetch from 'node-fetch'
-global.fetch = fetch
+import fetch from 'node-fetch';
+global.fetch = fetch;
```

Comment:
Maybe we can do something like this to make sure we only do this when we need to. Some versions of node already have this, and if the developer wants to use that version of `fetch`, then we shouldn't overwrite that version

```suggestion
import fetch from 'node-fetch';

if (typeof fetch === 'undefined') {
    global.fetch = fetch;
}
```

Reply:
ESLint is complaining:
<img width="1198" alt="image" src="https://github.com/mattermosttest/mattermost-plugin-e2e-test-utils/assets/2154092/112622f3-94da-4f88-8e6a-0130d0873d26">

Is this code correct? 😄 

If `fetch` is undefined, probably we don't want to use it to replace global?

Since the fetch reference here is from the import.

Reply:
@mickmister any thoughts?

Reply:
Maybe try this:

```ts
if (typeof fetch === 'undefined') {
    (global as any).fetch = fetch;
}
```